examples/mysql-wordpress-pd/README.md

```diff
@@ -278,6 +326,10 @@ NAME        CLUSTER-IP     EXTERNAL-IP     PORT(S)   AGE
 wordpress   10.0.0.5       1.2.3.4         80/TCP    19h
 ```
 
+**Note:** if you're using Host Path for storage then the wordpress service's
+`LoadBalancer` type is not implemented, and thus the wordpress pod's external
+IP address is not available (remains "Pending").
+
 ## Visit your new WordPress blog
```

Comment:
The docs are [here](http://kubernetes.io/docs/user-guide/services/#type-loadbalancer). Yes, `LoadBalancer` and volume support are both dependent on the environment external to kubernetes, however they are orthogonal. 

See the last bullet [here](https://github.com/kubernetes/kubernetes/blob/master/examples/guidelines.md#up-front). We would rather have the example assume a particular capability and then provide links to documentation to do it another way (Node port), rather than have every example attempt to support every environment. Examples get really hard to follow when there are so many if/then's in the flow.